### PR TITLE
Wall API

### PR DESCRIPTION
**Pull request by lknittel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,18 @@ Fork this repository, and submit your pull request providing documentation to th
 
 ### Product Specification
 * See wireframe.png
+
+### Requirements
+* Local mysql with user root
+* Create database 'wall'
+
+### Run API
+* Load project in IntelliJ
```
Comment (latch2112) [inline README.md:26]: check

File: README.md [hunk 2]
```diff
@@ -17,3 +17,18 @@ Fork this repository, and submit your pull request providing documentation to th
...
+* Execute "sbt run"
```
Comment (latch2112) [inline README.md:29]: [error]                  ^
[error] /Users/flachance/src/coding-challenge-fullstack/src/main/scala/org/crstejustine/wall/authentication/TokenConfig.scala:3:77: not found: type JwtHmacAlgorithm
[error] case class TokenConfig(tokenExpiration: Long, secretKey: String, algorithm: JwtHmacAlgorithm)
[error]                                                                             ^
[error] /Users/flachance/src/coding-challenge-fullstack/src/main/scala/org/crstejustine/wall/authentication/GenerateToken.scala:9:11: not found: value Jwt
[error]     Token(Jwt.encode(claims.expiresIn(config.tokenExpiration), config.secretKey, config.algorithm))
[error]           ^
[error] /Users/flachance/src/coding-challenge-fullstack/src/main/scala/org/crstejustine/wall/authentication/ValidateToken.scala:9:41: not found: type JwtClaim
[error]   override def apply(token: Token): Try[JwtClaim] =
[error]                                         ^
[error] 141 errors found
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 3 s, completed 29-Jul-2019 10:29:39 AM
crhsj105815:coding-challenge-fullstack flachance$

Comment (lknittel) [reply README.md:29]: Just updated the source, seems Optimize import have deleted all imports ... 
+ Add command in readme to load dependencies : "sbt clean update"

Comment (latch2112) [reply README.md:29]: 2019-07-29 12:55:21 -0400 - [ERROR] - com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Error thrown while acquiring connection from data source
java.sql.SQLException: (conn=9) Unknown system variable 'tx_isolation'
        at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:198)
        at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.getException(ExceptionMapper.java:110)
        at org.mariadb.jdbc.MariaDbStatement.executeExceptionEpilogue(MariaDbStatement.java:228)
        at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:334)
        at org.mariadb.jdbc.MariaDbStatement.executeQuery(MariaDbStatement.java:505)
        at org.mariadb.jdbc.MariaDbConnection.getTransactionIsolation(MariaDbConnection.java:859)
        at com.zaxxer.hikari.pool.PoolBase.checkDefaultIsolation(PoolBase.java:471)
        at com.zaxxer.hikari.pool.PoolBase.checkDriverSupport(PoolBase.java:434)
        at com.zaxxer.hikari.pool.PoolBase.setupConnection(PoolBase.java:402)
        at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:355)
        at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
        at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:473)
        at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71)
        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:727)
        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:713)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.SQLException: Unknown system variable 'tx_isolation'
Query is: SELECT @@tx_isolation
        at org.mariadb.jdbc.internal.util.LogQueryTool.exceptionWithQuery(LogQueryTool.java:119)
        at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:199)
        at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:328)
        ... 15 common frames omitted

Comment (lknittel) [reply README.md:29]: Maybe a pb with mysql version, added a docker to simplify the test
+ fix an error on username name used to test authentication

**Pull request closed without merging**